tony-core/src/main/java/com/linkedin/tony/Utils.java

```diff
@@ -416,5 +421,24 @@ public static void printWorkerTasksCompleted(AtomicInteger completedWTasks, long
     LOG.info("Completed worker tasks: " + completedWTasks.get() + " out of " + totalWTasks + " worker tasks." );
   }
 
+  public static Configuration addResourceToConfig(Configuration tonyConf, CommandLine cliParser) {
```

Comment:
Per discussion in https://github.com/linkedin/TonY/pull/103#pullrequestreview-178513945, let's make this a private method in `TonyClient` instead.

Reply:
👍